person.rb

```diff
@@ -19,6 +20,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Here we will do the same thing as in a book class. The rental belongs to a person.
```ruby
  def add_rental(book, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, book)
  end
```

The self refers to the person instance

Reply:
But also I think there is no contradiction between the two. We can/should have both. Right?

Reply:
@ahangarha, you are right, looking at the tutorial we can say a rental is an object (what you wrote) which makes sense. I will take a closer look at the tutorial once more.

Reply:
I think you were right. I have added the changes and now I think it looks good